airbyte-cdk/java/airbyte-cdk/core/src/test/java/io/airbyte/cdk/integrations/base/AirbyteLogMessageTemplateTest.java

```diff
@@ -88,7 +88,7 @@ public void testAirbyteLogMessageFormat() throws java.io.IOException {
     // validate that the message inside AirbyteLogMessage matches the pattern.
     // pattern to check for is: LOG_LEVEL className(methodName):LineNumber logMessage
     final String connectorLogMessageRegex =
-        "^INFO [\\w+.]*.AirbyteLogMessageTemplateTest\\(testAirbyteLogMessageFormat\\):\\d+ hello$";
+        "^INFO " + Thread.currentThread().getName() + " [\\w+.]*.AirbyteLogMessageTemplateTest\\(testAirbyteLogMessageFormat\\):\\d+ hello$";
```

Comment:
Call `Pattern.quote` on the thread name to escape it properly for use in a regex.